test/packages/good_input/_dev/test/config.yml

```diff
@@ -0,0 +1,4 @@
+system:
+  parallel: true
+policy:
+  parallel: false
```

Comment:
Have you considered having one file per test directory? I guess it depends on what is going to read this config. If it is going to be read by the commands it is fine here, if it is going to be read by the runners, I think it would be better in each test directory. But not a strong opinion.

Reply:
These files should be read/parsed before triggering all tests.
Currently, this means to be read in the command definitions. There is where the tests are triggered.

There is an option to move all the logic to get all tests to each runner. If so, then these files could be read in each runner before triggering any test.